src/Dashboard/README.md

```diff
@@ -63,4 +63,7 @@ directive:
       verb: New
       subject: Grafana
     hide: true
+  # The cmdlet's name to long, Re-name it
+  # - model-cmdlet:
+  #     - AzureMonitorWorkspaceIntegration
```

Comment:
If the commented code will not be used again, consider removing.

Reply:
After a command is renamed, the command name is different from the original one. Therefore, retain comments for subsequent maintenance